libs9s/s9soptions.cpp

```diff
@@ -3241,6 +3244,27 @@ S9sOptions::upgradeMethod() const
     return getString("upgrade_method");
 }
 
+/**
+ * \returns  true if the old node data after major upgrade should be deleted.
+ */
+bool
+S9sOptions::deleteOldNode() const
+{
+    return getBool("delete_old_node");
+}
+
+int
+S9sOptions::upgradeTmpPort() const
+{
+    return getString("upgrade_tmp_port").toInt();
```

Comment:
If field is not present, it should be returning 0. I think it should use default value

Reply:
The default should be left for cmon backend to be picked. That is why 0 is used here as default. It is kind of incalid that can be recogised on the backend side.